src/XenobladeChroniclesX/Mods/CharacterSize/rules.txt

```diff
@@ -0,0 +1,445 @@
+[Definition]
+titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
+name = Character Size and Invisible Chest
+path = "Xenoblade Chronicles X/Mods/Character/Character Size"
+description = "You can change the height of any character and also make the chest of some characters not render.||Recomended to be combined with the 'No Fading When Close-up' mod (in Graphics).||***Activate by pressing START(+), then Party->Active Members->Confirm Changes."
```

Comment:
```suggestion
description = "You can change the height of any character and also make the chest of some characters not render.||Recommended to be combined with the 'No Fading When Close-up' option (in Graphics).||***Activate by pressing START(+), then Party->Active Members->Confirm Changes."
```

Reply:
fixed the spelling mistake of recommended
I think mod is a better word than option because it is a mod? idk, maybe thats just me
option could maybe be confused for an ingame option or something